datafusion/sqllogictest/test_files/select.slt

```diff
@@ -1688,8 +1688,14 @@ physical_plan
 
 # due to the reason describe in https://github.com/apache/datafusion/issues/8927,
 # the following queries will fail
```

Comment:
Since the test below now works, I think we could move this comment to the test below this one.